README.md

```diff
@@ -39,3 +49,61 @@ Before proceeding, make sure that there are no current builds in the `Build` ste
+
+Before proceeding, make sure that there are no current builds in the `Build` step of the [prod pipeline](https://cloud.ibm.com/devops/pipelines/fcc7c3e9-9c40-4a58-8a7f-09c08413ab7d?env_id=ibm:yp:us-south). This is so merging pull requests later into prod does not impact any current deployments.
+
+1. Once all of the changes look correct on the [staging site](https://staging-openlibertyio.mybluemix.net/docs/), and all of the `staging` branch changes have been merged into the `vNext` branch, create the branches you need for the new release
+   a. Select `vNext` in the **Current Branch** menu in Github Desktop
+   b. Click the **New Branch** button and create a branch with the next release version number. E.g. if the most recent version on openliberty.io is `20.0.0.8`, then create a `v20.0.0.9`  branch. 
+
+2. Follow the same procedure in steps 1a-1b to create `vX.0.0.X-staging` and `vX.0.0.X-draft` branches off of the new version branch you just created, which will be used for retroactive fixes to this version in the future. E.g. if `v20.0.0.9` is the version being released, then select `v20.0.0.9` from the **Current Branch** menu in Github desktop and create a new branch called v20.0.0.9-staging`  Switch back to  `v20.0.0.9` branch and repeat to create v20.0.0.9-draft`. **Don't forget to publish the branch all the brannches that you create by clicking "Publish this branch" after you create each branch**.
+  
+3. Repeat steps 1 and 2 for the [docs-generated repo](https://github.com/OpenLiberty/docs-generated).
+
+4. Create a new branch off of the `prod` branch of this repo to add the new release version to the `branches` section of the [`prod playbook`](https://github.com/OpenLiberty/docs-playbook/blob/prod/antora-playbook.yml). You can make this change in your browser.
+   a. Go to the [antora-playbook.yml](https://github.com/OpenLiberty/docs-playbook/blob/prod/antora-playbook.yml). 
+   b. Click the pencil icon to edit the file. 
+   c. Add the new release version to the `branches` section
+   d. After you edit the file, create a pull request into the `prod branch` by clicking the **Propose changes** button. Once the pull request has been reviewed, merge it in.
+
+5. Repeat same procedure in steps 4b-4d to change the `version` in `antora.yml` in the [`staging`](https://github.com/OpenLiberty/docs/blob/staging/antora.yml) and [`draft`](https://github.com/OpenLiberty/docs/blob/draft/antora.yml) branches of the docs repo to the next release `vX.0.0.X` version incremented from the current release being published to openliberty.io. Once those changes have been reviewed merge them in. This is important so that the staging and draft builds can complete. Antora requires all of the versions of `antora.yml` of the branches being used in a build to have a unique version so they need to be updated to be unique from the version just released. You can either change the version and commit to draft/staging or create a pull request to do so.
```

Comment:
I would expect this to potentially cause merge issues since this is creating different commits in the history of staging and draft for this file.  I would expect the process for updating the antora.yml files to follow the [documented process](https://github.com/OpenLiberty/docs#updating-an-existing-topic) for delivering anything into the docs repo (aka, create and deliver a PR from a 'personal' branch into draft and staging).  The only exception would be the nav.adoc file.

Reply:
It does create different commits in draft and staging history- but does not result in conflicts because draft is never merged into staging in the docs repo. Doing so would always result in conflicts and files being prematurely sent to prod.

This is the process I have always used for the antora.yaml- but you're right, it is outside the documented process and maybe better not to document. We are "road-testing" these instructions internally with the 22.0.0.6 release so I'll try it the official way instead and update this section accordingly.